README.md

```diff
@@ -4,3 +4,44 @@ This is a Python 3 port of the utility functions from Dragon Research Labs
 [RPKI Toolkit](https://github.com/dragonresearch/rpki.net), also known as
 "rpki.net".  This is intended as a proof of concept / feasibility study more
 than anything else.
+
+Installation and Usage instructions.
+
+```shell
+$ sudo git clone https://github.com/aftabsiddiqui/py3-rpki-utils.git
+$ cd py3-rpki-utils
+$ sudo python3 setup.py build
+$ cp build/*/*/*/_POW*.so rpki/POW/_POW.so
+$ cd print
```

Comment:
Any particular reasons for the `sudo`?  This should not require privilege.

Reply:
oh thats privilege issue was on my local system, I over looked that. No need for sudo for any of these operations.